src/MoBi.Core/Commands/ChangeValueFormulaCommand.cs

```diff
@@ -63,21 +61,54 @@ protected override void ExecuteWith(IMoBiContext context)
       {
          base.ExecuteWith(context);
          _changedStartValue.Formula = _newFormula;
-         Path = _changedStartValue.Path;
+         Path = PathFrom(_changedStartValue);
       }
 
-      protected override ICommand<IMoBiContext> GetInverseCommand(IMoBiContext context)
-      {
-         return new ChangeStartValueFormulaCommand<T>(_startValuesBuildingBlock, _changedStartValue, _oldFormula, _newFormula).AsInverseFor(this);
-      }
+      protected abstract IObjectPath PathFrom(T changedStartValue);
```

Comment:
The reason for it is that IObjectPath Path is defined in IStartValue, but I don't want to use that abstraction here because I did have that, but it led to confusion when I was creating the commands. Eg. you pull some stuff up that relies on object.StartValue and it compiles and seems ok, until you go in the UI and try to change the formula. In the end I decided it wasn't worth the risk of confusion.

I created an issue to collect these for later. 
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/1811